nautobot/dcim/filters.py

```diff
@@ -712,7 +712,7 @@ def _device_bays(self, queryset, name, value):
         return queryset.exclude(devicebaytemplates__isnull=value)
 
 
-# TODO: should be DeviceTypeComponentFilterSetMixin
+# 2.0 TODO: should be DeviceTypeComponentFilterSetMixin
```

Comment:
We should introduce these renames in 1.5 and deprecate the old names, per #2530 we would then be able to remove the old names entirely in 2.2.

Reply:
Another great point. Thank you.